### ISSUE TITLE
Embedding model to inference mode (llm 01L)

### ISSUE THREAD

**Issue by patputnam-db** [databricks-academy/llm-foundation-models#5]
The TransformerEncoder model in CMD 14 is instantiated in 'training' mode. As such, the dropout layers are not frozen during inference. As such, re-running the embedding model multiple times on the same input tensors results in different embeddings.

Fix: Add a model.eval() call after the model is instantiated (cmd 14, line 11).
